libs/partners/databricks/pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[tool.poetry]
+name = "langchain-databricks"
+version = "0.1.0"
+description = "An integration package connecting Databricks and LangChain"
+authors = []
```

Comment:
should we fill in authors?

Reply:
Hmm ideally yes, probably it's tedious to maintain the list. Most of other partner packages leave this blank.